### PR TITLE
Fix documentation build

### PR DESCRIPTION
**Pull request by rexwangcc**
Now we already have documentation on readthedocs on my branch: http://pipeline-tools.readthedocs.io/en/cw-add-autodoc/ , this PR fixes the build on master branch.

Please ensure the following when opening a PR:
- [ ] Added or updated tests
- [x] Updated documentation
- [x] Applied style guidelines
- [x] Considered generalizability beyond our own use case

### CODE REVIEW

File: pipeline_tools/dcp_utils.py [hunk 1]
```diff
@@ -79,7 +82,9 @@ def get_auth_token(url="https://danielvaughan.eu.auth0.com/oauth/token",
                    grant_type="client_credentials"):
     """Request and get the access token for a trusted client from Auth0.
 
-    Note: The parameters and credentials here are meant to be hard coded, the authentication is purely for identifying a user it doesn't give and permissions.
+    .. note::
+
+        The parameters and credentials here are meant to be hard coded, the authentication is purely for identifying a user it doesn't give and permissions.
```
Comment (samanehsan) [inline pipeline_tools/dcp_utils.py]: Should this say "it doesn't give any permissions"?

**Comment by samanehsan** [comment]
I have a comment about one of the docstrings, otherwise this looks good to me!

File: pipeline_tools/dcp_utils.py (hunk 1 above)
Comment (rexwangcc) [reply pipeline_tools/dcp_utils.py]: Yes, it was a typo 🐛 Nice catch!

**Comment by rexwangcc** [comment]
@samanehsan Would you mind taking another look here 😺

**Comment by rexwangcc** [comment]
@samanehsan Thanks!

**Pull request merged**